### DISCUSSION TITLE
Alternative hardware for high refresh rates

### DISCUSSION THREAD

**Discussion by devRomeos** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#596] [Q&A] [answered]
Hello, I am studying the best option for my project, I plan to use only a single 32x32 panel , but 3840hz and I want to avoid the black bars when you take a daytime picture, also the "dizziness" when you turn your head quickly.

What I learned here it's that esp32 it's way too far to archve that refresh, even for a single  color . Not sure if this is the place to ask this but I wonder if there is out there any other hardware or card that can handle my needs, just want to show static 4-5 bits images, and change it on demand 

any feedback is welcome

**Comment by mrcodetastic** [comment]
3840hz will need a fpga or a super low colour depth

**Reply by devRomeos** [reply]
Hum, do you think I can get atleast ~1500hz  with 4 bits images? I had tried the [spreadsheet calculator](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/blob/master/doc/memcalc.xlsm)  but the refresh rate it's always stuck on 143hz

**Reply by mrcodetastic** [reply, marked as answer]
Ckock speed in hz (Mhz)  / 32 / 32 / colordepth^2

Should = refresh rate

**Reply by mrcodetastic** [reply]
Work backwards from that

**Reply by devRomeos** [reply]
Gotcha, thanks !
Asuming 20Mhz with 32 x 32 total pixel at 3 Bit I get 2170hz that might be  enough 

Anyway, if some body know any other hardware or chinese card where I can do this in a more fast way let me know, I only need to push an static image from time to time, sending the change command by UART or something

**Reply by mrcodetastic** [reply]
Actually, I might be wrong,  what is the "scan type" of the panel?

1/16? Which means two parallel scan lines at the same time. 

If so,  then one of those 32's in my equation should be a 16.

So double the refresh rate.

**Reply by devRomeos** [reply]
I don't have panels on my own yet as I'm researching before take action, the panel I am considering are 1/4 and 1/6, does that mean it would be double the double? 😚

**Reply by mrcodetastic** [reply]
This library only supports two scan panels. Check out the readme.

**Reply by srcastor** [reply]
Can you elaborate that please? In the readme seems to support 1/4 and 1/8 

![image](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/12575586/2840a1d1-0b96-4568-b39e-38dfe0ba1805)

**Reply by mrcodetastic** [reply]
We have a hack using the virtual matrix class, but it is not guaranteed. Electrically the library is coded for 1/16 or 1/32, 2 scan panels.

**Reply by hijoh** [reply]
When I take photos, the display screen will have black blocks. How to solve the problem?
